src/Core/Manifest/VersionProvider.php

```diff
@@ -180,16 +180,23 @@ public function getModules()
     /**
      * Tries to obtain version number from composer.lock if it exists
      *
-     * @param array $modules
-     * @return array
+     * @param array<int,string> $modules
+     * @return array<int|string,mixed>
```

Comment:
```suggestion
     * @return array<string|string>
```
`$versions[$module]` is gonna add a string key, not an int key.
Where does the `mixed` come from? That seems incorrect?